comps/llms/deployment/docker_compose/compose_faq-generation.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+include:
+  - ../../../third_parties/tgi/deployment/docker_compose/compose.yaml
+  - ../../../third_parties/vllm/deployment/docker_compose/compose.yaml
+
+services:
+  llm-faqgen-server:
+    image: opea/llm-faqgen:latest
+    container_name: llm-faqgen-server
+    ports:
+      - ${FAQ_PORT:-9000}:9000
+    ipc: host
+    environment:
+      no_proxy: ${no_proxy}
+      http_proxy: ${http_proxy}
+      https_proxy: ${https_proxy}
+      LLM_ENDPOINT: ${LLM_ENDPOINT}
+      LLM_MODEL_ID: ${LLM_MODEL_ID}
+      HUGGINGFACEHUB_API_TOKEN: ${HUGGINGFACEHUB_API_TOKEN}
+      FAQGen_COMPONENT_NAME: ${FAQGen_COMPONENT_NAME}
+      LOGFLAG: ${LOGFLAG:-False}
+    restart: unless-stopped
+
+  llm-faqgen-server-tgi:
+    extends: llm-faqgen-server
+    container_name: llm-faqgen-server-tgi
+    depends_on:
+      tgi-server:
```

Comment:
the same container name (e.g. tgi-server) may cause conflict for different concurrent CI flows on one machine. Do we aim to solve this in this PR?

Reply:
suyue metioned there will only be only one CI triggered for one repo.